app/containers/SongContainer/index.js

```diff
@@ -0,0 +1,250 @@
+import { Card, Skeleton, Input } from 'antd';
+import styled from 'styled-components';
+import { injectIntl } from 'react-intl';
+import { injectSaga } from 'redux-injectors';
+import { selectSongContainer, selectSongsData, selectSongsError, selectQuery } from './selectors';
+import { songContainerCreators } from './reducer';
+import songContainerSaga from './saga';
+import T from '@components/T';
+
+
+const { Search } = Input;
+const { Meta } = Card;
+
+const CustomCard = styled(Card)`
+  && {
+    width: ;
+    display: flex;
+    flex-wrap: wrap;
+    color: ${props => props.color};
+    ${props => props.color && `color: ${props.color}`};
+  }
+`;
+
+const SongCard = styled(Card)`
+  && {
+    margin: 1em;
+    
+  }
+`;
+
+const ImageArt = styled.img`
+&& {
+    object-fit: cover;
+    height: 250px
+}
+`
+
+const Container = styled.div`
+  && {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    max-width: ${props => props.maxwidth}px;
+    padding: ${props => props.padding}px;
+  }
+`;
+const ResultContainer = styled.div`
+  && {
+    display: flex;
+    flex-direction: column;
+    max-width: 80vw;
+    width: 100%;
+    margin: 20px auto;
+    padding: ${props => props.padding}px;
+  }
+`;
+const RightContent = styled.div`
+  display: flex;
+  align-self: flex-end;
+`;
+
+const useProgressiveImg = (lowQualitySrc, highQualitySrc) => {
+  const [src, setSrc] = React.useState(lowQualitySrc);
+  React.useEffect(() => {
+    setSrc(lowQualitySrc);
+    const img = new Image();
+    img.src = highQualitySrc;
+    img.onload = () => {
+      setSrc(highQualitySrc);
+    };
+  }, [lowQualitySrc, highQualitySrc]);
+  return [src, { blur: src === lowQualitySrc }];
+};
+
+export function SongContainer({
+    dispatchSongs,
+    dispatchClearSongs,
+    intl,
+    songsData = {},
+    songsError = null,
+    query,
+    maxwidth,
+    padding
+}) {
+
+    const [loading, setLoading] = useState(false);
+
+
+
+    useEffect(() => {
+        const loaded = get(songsData, 'results', null) || songsError;
+        if (loading && loaded) {
+            setLoading(false);
+        }
+    }, [songsData])
+
+    useEffect(() => {
+        if (query!='' && songsData?.songs?.length) {
+            setLoading(true)
+            dispatchSongs(query);
+        } 
+        else {
+            dispatchClearSongs()
+            setLoading(false)
+        }
+    }, []);
+    const handleOnChange = rName => {
+        if (!isEmpty(rName)) {
+            dispatchSongs(rName);
+            setLoading(true);
+
+        } else {
+            dispatchClearSongs();
+        }
+    };
+    const debouncedHandleOnChange = debounce(handleOnChange, 200);
+
+    const refreshPage = () => {
+        history.push('stories');
+        window.location.reload();
+    };
+
+    const AlbumArt = ({source}) => {
+        const [src, { blur }] = useProgressiveImg(`https://via.placeholder.com/250x250/ffffff/808080/?text=loading`, source);
```

Comment:
Love what you've done here! 
We get multiple resolutions for artwork, lets use the lowest one instead of `https://via.placeholder.com/250x250/ffffff/808080/?text=loading``